README.md

```diff
@@ -1,3 +1,53 @@
+- Getting started with ship-secure-frontend project
+
+# Cloning
+
+From https://github.com/Snowwhite-Workshop/ship-secure-frontend copy the url under code section
+
+Go to your command prompt or terminal, navigate to the folder you wish to work on and type the following command
+
+$ git clone THE_URL_YOU_COPIED_FROM_GITHUB (ensure that your internet is up). Then,
+
+$ cd ship-secure-frontend/ (the newly cloned folder)
+
+# Install project dependencies
+
+$ npm install (or yarn install - if you are using yarn)
```

Comment:
everyone should use npm only..
did you use yarn or npm when you were creating the base files?

Reply:
Noted. I used npm